src/ClientData/ScopeIdProvider.cpp

```diff
@@ -46,10 +46,9 @@ std::unique_ptr<NameEqualityScopeIdProvider> NameEqualityScopeIdProvider::Create
     scope_id_to_info.try_emplace(scope_id, scope_info);
     scope_info_to_id.try_emplace(scope_info, scope_id);
     scope_id_to_function_info.try_emplace(
-        scope_id,
-        FunctionInfo{instrumented_function.file_path(), instrumented_function.file_build_id(),
-                     instrumented_function.function_virtual_address(),
-                     instrumented_function.function_size(), instrumented_function.function_name()});
+        scope_id, /* piecewise_construct FunctionInfo */ instrumented_function.file_path(),
```

Comment:
I'm afraid, `piecewise_construct` isn't the correct term here. `piecewise_construct` refers to e.g. constructing both key and the value. Here I'd rather say, we construct FunctionInfo in-place. So I suggest to replace the comment with "in-place FunctionInfo construction".

Reply:
Done